site/gatsby-site/src/components/layout/Footer.js

```diff
@@ -48,6 +48,7 @@ export default function Footer() {
                 url {
                   url
                 }
+                path
```

Comment:
Why do we need to add the `path` field? The `RSS` item is using the `path` field or the `url` field?

Reply:
RSS is using the path because it's not an external URL